### ISSUE TITLE
does this project needs/wants donations?

### ISSUE THREAD

**Issue by pedro-nonfree** [coturn/coturn#766]
does this project needs/wants donations? Independently about the response, I would like to see that info more visible. coturn project is great

**Comment by misi** [comment]
Absolutely. We need donation of engineering time so please donate persons who could improve or bug fix issues.
We can collect money to hire engineers too, but then we need to setup such infrastructure.
